srcpkgs/ibus-m17n/INSTALL

```diff
@@ -0,0 +1,5 @@
+case "$ACTION" in
+post)
+	ibus-daemon -xdr
```

Comment:
:thinking: Assuming the desktop is not running as `root`, this looks like it just starts another `ibus-daemon` as `root`. This instance might not be functional/accessible in the user's session and any settings changes made to it will likely appear to be gone on the next login because they are saved as `root`. Are you sure this is what you want?

Reply:
I never thought of it that way. Since `xbps-install` is being run with sudo, it will run the command as root.

I don't know if there is a way to run the command as a normal user. Every solution I found needed a username.

Do you have any suggestion? Or should we remove the INSTALL file and let the user run the command (only INSTALL.msg)?

Reply:
I'm not aware of a robust way to restart all `ibus-daemon`s in the system, but personally, I wouldn't even try. It would probably interfere with users typing through IBus while `ibus-m17n` is being installed/updated. It might even crash their application. I'd just drop the `INSTALL` altogether, `INSTALL.msg` is plenty informative.

Reply:
I have removed `INSTALL`. So I guess everything should be ok now.